src/invariant/BucketListIsConsistentWithDatabase.cpp

```diff
@@ -134,6 +135,9 @@ BucketListIsConsistentWithDatabase::checkOnBucketApply(
                 case DATA:
                     ++nData;
                     break;
+                case CLAIMABLE_BALANCE:
+                    ++nClaimableBalance;
```

Comment:
you forgot to actually check against the database using `countObjects` (line 185)

Reply:
Ah good catch. Fixed.